holmes/plugins/toolsets/quickwit/quickwit.py

```
@@ -0,0 +1,255 @@
+    @property
+    def quickwit_config(self) -> QuickwitConfig:
+        return self.config  # type: ignore[return-value]
+
+    def _health_check(self) -> Tuple[bool, str]:
+        cfg = self.quickwit_config
+        url = f"{cfg.api_url.rstrip('/')}/health/livez"
+        try:
+            response = requests.get(url, timeout=min(cfg.timeout_seconds, 10))
+            response.raise_for_status()
+            return True, f"Connected to Quickwit at {cfg.api_url}"
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)